fileserver/routes.py

```diff
@@ -345,24 +353,39 @@ def get_session_version():
 
         updated = row[0]
 
-        # Fetch the latest release version
-        cur.execute(
-            """
-            SELECT id, version, name, notes from release_versions
-            WHERE proj_name = %s ORDER BY version_code DESC""",
-            (project,),
-        )
+        if release_channel == "alpha":
+            # Fetch the latest alpha release version
+            cur.execute(
+                """
+                SELECT id, version, name, notes, alpharelease from alpharelease_versions
+                WHERE proj_name = %s ORDER BY version_code DESC""",
+                (project,),
+            )
+        else:
+            # Fetch the latest release version
+            cur.execute(
+                """
+                SELECT id, version, name, notes, alpharelease from release_versions
+                WHERE proj_name = %s ORDER BY version_code DESC""",
+                (project,),
+            )
 
         row = cur.fetchone()
         if row is None:
             app.logger.warn("{} has no releases!".format(project))
             return error_resp(http.BAD_GATEWAY)
-
+        
         release_id = row[0]
+        release_version =  row[1]
+        alpha_build_number = row[4]
+
+        if alpha_build_number is not None:
+            release_version += f"-alpha.{alpha_build_number}"
```

Comment:
Perhaps in the `versions` view we could change the `version` column from so that this is reflected directly in the view, rather than having to be manipulated here in the code:
```
version_code / 1000000 || '.' || version_code % 1000000 / 1000 || '.' || version_code % 1000 AS version,
```
to:
```
version_code / 1000000 || '.' || version_code % 1000000 / 1000 || '.' || version_code % 1000 ||
    (CASE WHEN alpharelease IS NOT NULL THEN '-alpha.' || alpharelease ELSE '')
    AS version,
```